eq/pipe.cpp

```diff
@@ -326,6 +394,16 @@ WindowSystem Pipe::selectWindowSystem() const
 #elif WGL
     return WindowSystem( "WGL" );
 #elif EQ_QT_USED
+    if( !pipe.isWindowSystemAvaiable( "Qt" ))
```

Comment:
That code path is used only if none of AGL, GLX or WGL are defined, which is pretty rare.